app/models/user.rb

```diff
@@ -14,6 +14,7 @@ class User < ApplicationRecord
   before_save :downcase_email
 
   scope :get_trainees, ->{select(:id, :full_name).where role_id: :trainee}
+  scope :get_supervisors, ->{select(:id, :full_name).where role_id: :supervisor}
```

Comment:
16,17 chỉ cần viết scope là
```suggestion
  scope :get_name, ->{select(:id, :full_name)}
```
còn sẽ gọi theo kiểu enum e nhé. 
vd: User.admin -> danh sách admin
User.supervisor
-> cái này enum hỗ trợ, e tìm hiểu thêm nhé

Reply:
Vâng ạ.